DevCycle/Models/Cache.swift

```diff
@@ -72,7 +118,19 @@ class CacheService: CacheServiceProtocol {
         return defaults.string(forKey: key)
     }
     
+    private func setInt(key: String, value: Int) {
+        defaults.set(value, forKey: key)
+    }
+    
+    private func getInt(key: String) -> Int? {
+        return defaults.integer(forKey: key)
+    }
+    
     private func remove(key: String) {
         defaults.removeObject(forKey: key)
     }
+    
+    private func getKeyPrefix(user: DVCUser) -> String {
```

Comment:
getConfigKeyPrefix could be more clear?
```suggestion
    private func getConfigKeyPrefix(user: DVCUser) -> String {
```